app/packages/core/src/plugins/SchemaIO/components/TableView.tsx

```diff
@@ -47,23 +47,26 @@ export default function TableView(props: ViewPropsType) {
   const selectedCellColor =
     selected_color || theme.palette.background.activeCell;
 
-  const getRowActions = useCallback((row) => {
-    const computedRowActions = [] as any;
-    for (const action of row_actions) {
-      if (action.rows?.[row] !== false) {
-        computedRowActions.push({
-          ...action,
-          onClick: (action, e) => {
-            handleClick(panelId, {
-              operator: action.on_click,
-              params: { path, event: action.name, row },
-            });
-          },
-        });
+  const getRowActions = useCallback(
+    (row) => {
+      const computedRowActions = [] as any;
+      for (const action of row_actions) {
+        if (action.rows?.[row] !== false) {
+          computedRowActions.push({
+            ...action,
+            onClick: (action, e) => {
+              handleClick(panelId, {
+                operator: action.on_click,
+                params: { path, event: action.name, row },
+              });
+            },
+          });
+        }
       }
-    }
-    return computedRowActions;
-  }, []);
+      return computedRowActions;
+    },
+    [row_actions, handleClick, panelId, path]
+  );
 
   const getTooltips = useCallback((tooltipList) => {
```

Comment:
Do we need to do the same for this function too?

Reply:
seems to work fine with my local testing